modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,31 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `Subscription` object
 
 .Procedure
 
-. Run the following command to patch the CSV:
+* Increase memory limits to `800Mi` by editing the `Subscription` object:
 +
-[source,terminal]
+[source,yaml]
 ----
-$ oc -n netobserv patch csv network-observability-operator.v1.0.0 --type='json' -p='[{"op": "replace", "path":"/spec/install/spec/deployments/0/spec/template/spec/containers/0/resources/limits/memory", value: "1Gi"}]'
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: netobserv-operator
+  namespace: openshift-netobserv-operator
+spec:
+  channel: stable
+  config:
+    resources:
```

Comment:
Thanks @jotak . I sorta flipped your points, and added the "For example, you can increase the limit to 800Mi" as a code callout and added the `spec.config.resources.limits.memory` to the Procedure intro.

Reply:
I see why we want to encourage the subscription method due to the upgrade case, but I feel like we shouldn't remove the one-liner command altogether, or at least sub-in an alternate command that can do this? e.g. `oc edit...` 

Maybe some might consider this too verbose but I also think we can state the case of the change, e.g. `You can configure limits via the Sub so that they persist through future operator updates` or something like that

Reply:
I added an alternate command...I can agree the one-line command is a nice-to-have vs the web console. Even though we don't have CLI alternatives in all our examples, in this case, its a really straightforward CLI command, so I can see adding it is a +.